archimedes/src/app/app.component.html

```diff
@@ -15,8 +15,37 @@
             </app-search-box>
         </div>
     </div>
+    <div class="row">
+        <div class="col">
+            <label class="custom-control custom-checkbox">
+                <input type="checkbox" class="custom-control-input" name="program"
+                    [(ngModel)]="selected.program" (change)="SearchBox.toggle('programs', selected.program)">
+                <span class="custom-control-indicator"></span>
+                <span class="custom-control-description">Programs</span>
+            </label>
+            <label class="custom-control custom-checkbox">
+                <input type="checkbox" class="custom-control-input" name="news"
+                    [(ngModel)]="selected.news" (change)="SearchBox.toggle('news', selected.news)">
+                <span class="custom-control-indicator"></span>
+                <span class="custom-control-description">News</span>
+            </label>
+            <label class="custom-control custom-checkbox">
+                <input type="checkbox" class="custom-control-input" name="image"
+                    [(ngModel)]="selected.image" (change)="SearchBox.toggle('images', selected.image)">
+                <span class="custom-control-indicator"></span>
+                <span class="custom-control-description">Images</span>
+            </label>
+        </div>
+    </div>
+    <div class="row" *ngIf="!selected.program && !selected.news && !selected.image">
+        <div class="col">
+            <div class="alert alert-info" role="alert">
+                Select at least one filter above to view results
```

Comment:
Minor edit: Add a period at the end of this in order to match the other alert messages.